Tasks/DownloadBuildArtifactsV0/task.json

```diff
@@ -8,7 +8,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 138,
+    "Minor": 140,
```

Comment:
I updated the version to 140 because of this pull request: [7502](https://github.com/Microsoft/vsts-tasks/pull/7502).
If there is a better way to do this let me know.

Reply:
Since M139 milestone is still going on, if this PR completes by end of this week or early next week, it is fine to keep the minor version as M139 only

Reply:
@lypolyzo - keeping milestone as 140 is fine as we are now into 140 sprint.

Reply:
changed to 141